examples/connection.rb

```diff
@@ -0,0 +1,27 @@
+require "yast"
+require "y2network/config"
+require "y2network/virtual_interface"
+require "y2network/connection_config/ethernet"
+require "y2network/connection_config/bond"
+require "y2network/connection_config/wireless"
+
+# given an interfaces list (interfaces)
+
+config = Y2Network::Config.from(:sysconfig)
+
+eth0 = config.interfaces.find("eth0").first
+eth_conn = Y2Network::ConnectionConfig::Ethernet.new
+eth_conn.interface = eth0
+
+eth1 = config.interfaces.find("eth1").first
```

Comment:
I do not get why find returns array and not single element. That is strange. Also it can find multiple interfaces with name "eth1" - it is strange.

Reply:
`interfaces.find { |i| i.name == "eth1" }`

Reply:
I updated the example